File: Python/menor_de_tres.py
p1 = int(input("Digite o primeiro valor: "))
p2 = int(input("Digite o segundo valor: "))
p3 = int(input("Digite o terceiro valor: "))

if p1 < p2 and p1 < p3:
    menor = p1
elif p2 < p3:
    menor = p2
else: menor = p3

print(f"MENOR = {menor}")

File: Java/Menor_de_tres.java
import java.util.Scanner;

public class Menor_de_tres {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int valor1, valor2, valor3, menor;

        System.out.print("Primeiro valor: ");
        valor1 = sc.nextInt();
        System.out.print("Segundo valor: ");
        valor2 = sc.nextInt();
        System.out.print("Terceiro valor: ");
        valor3 = sc.nextInt();

        if (valor1 < valor2 && valor1 < valor3) {
            menor = valor1;
        }
        else if (valor2 < valor3) {
            menor = valor2;
        }
        else {
            menor = valor3;
        }

        System.out.println("MENOR = " + menor);

        sc.close();
    }
}